README.md

```diff
@@ -1,11 +1,283 @@
+
+The sample projects are all set up to use the following test-only bundle ID: `com.aol.mobile.one.testapp`
+
+## High-Level Architecture Overview <a name="architecture"></a>
+At a high-level, the OMSDK architecture is composed of the following components:
+* SDK Core
+* A set of Video Renderers that are either built-in (e.g., flat 2D and RYOT 360°) or external plugins (e.g., Verizon Envrmnt 360°, custom, etc.)
+* O2 VRM (video rights management) VAST Ads Engine
+* Video and Ads Analytics module
+* Default video player controls UX implementation
+
+Our modular approach makes it easy for us (or you) to add new renderers in the future, or for you to add your own custom video player controls UX implementation. Under the hood, we rely on the built-in iOS [`AVPlayer`](https://developer.apple.com/documentation/avfoundation/avplayer) to handle the actual video playback.
+
+## How the SDK works <a name="how"></a>
+At a very basic level, the OMSDK controls only the video frame. Because of this, you are completely in control of your app’s design and UX (look and feel). You can control whether videos play in a small view, in-place over a thumbnail image, or at full-screen. Your app also has complete control over device rotation, use of view/navigation controllers, scrollers, and any transitions between them. The SDK does not dictate any overall visual design or behavior on your app.
+
+If you choose to use the SDK’s built-in default player controls UX implementation, then that part of the video UX is imposed on you. All controls rendering is also done within the frame you provide for the video. Regardless of which controls UX you use, we currently do not allow any customization or overriding of the ads playback UX (which is different from the normal video playback UX), so that visual interface is dictated, and you cannot override it. Future customization options are planned here.
+
+To play a video, you follow these very basic steps:
+1. Initialize an instance of the OMSDK
+2. Initialize a new `Player` object with a video ID
+3. Attach it to a view controller
+4. Play it
+
+That’s it!
+
+Behind the scenes, what happens is this … the initialization of an instance of the SDK takes your app’s bundle ID, and passes it to our back-end micro services to authenticate your application for video playback within the O2 video platform. The server passes back all the necessary playback and authentication information down to that instance of the SDK, for use during it’s lifespan. When you construct a new `Player` object from that SDK instance, it communicates with our micro services to obtain all the necessary video metadata (e.g., thumbnails and video URLs, duration, etc.). This Player`` object will play and replay the associated video until deinitialized.
```

Comment:
This is heavy details which are not required.

Reply:
I believe it fits ... it's still a generalization; but gives the tech reader a sense of what happens under the covers.

Reply:
Nobody needs this details without facts. We are closed sourced so under-the-hood happens a lot more - this information could become outdated very quickly.